content/components/tokens.mdx

```diff
@@ -23,99 +36,167 @@ import {LabelGroup, Label, Box} from '@primer/components'
-        src="https://user-images.githubusercontent.com/2313998/137778682-ee35f8e0-72de-4a24-b1bd-e4ea02e2ac49.png"
-    />
-    <Box as="ul" mt="0">
-        <Box as="li" mt="0"><strong>Leading visual</strong> (optional): Provides additional context (e.g.: user avatar)</Box>
-        <Box as="li"><strong>Label</strong> (required): Concise text label</Box>
-        <Box as="li"><strong>Remove button</strong> (optional): Removes the token from an editable group of tokens</Box>
+
+<Box
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 2}}
+>
+  <img
+    width="456"
+    alt="token anatomy diagram"
+    src="https://user-images.githubusercontent.com/2313998/137778682-ee35f8e0-72de-4a24-b1bd-e4ea02e2ac49.png"
+  />
+  <Box as="ul" mt="0">
+    <Box as="li" mt="0">
+      <strong>Leading visual</strong> (optional): Provides additional context (e.g.: user avatar)
+    </Box>
+    <Box as="li">
+      <strong>Label</strong> (required): Concise text label
     </Box>
+    <Box as="li">
+      <strong>Remove button</strong> (optional): Removes the token from an editable group of tokens
+    </Box>
+  </Box>
 </Box>
 
 ## Interactivity
+
 Tokens are almost always interactive. When a token is interactive, it will react to cursor hover and keyboard focus.
 
 While tokens may have the same interactivity as a button or a link, a token should not be used as a replacement for a button.
 
 If a token is a link or button _and_ it has a remove button, keyboard focus will only go to the link/button. Keyboard users can remove a token by focusing the link/button and pressing "Backspace" or "Delete". When a screen reader is focused on a removable token, it will read the token's label and tell users to press "Backspace" or "Delete" to remove it.
 
 Examples of interactions:
+
 - An issue label token that links to a list of issues with that label
 - A user token that shows a profile summary dialog on hover
 
 ## Sizing
 
-<Box display="flex" alignItems="flex-start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 4}}>
-    <img
-        width="456"
-        alt="tokens rendered at all sizes"
-        src="https://user-images.githubusercontent.com/2313998/137771748-ee4d796a-7bf0-4384-bb75-a239f358ede6.png"
-    />
-    <Box>
-        <Box as="p" mt="0">Different token sizes can be used to maintain visual harmony with the other content on the screen or to maintain a comfortable touch target for interactive tokens.</Box>
-        <Box as="p">Keep visual hierarchy in mind. Large tokens should not visually compete with elements that have a higher priority.</Box>
-        <Box as="p">A group of tokens should not have tokens of different sizes.</Box>
+<Box
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 4}}
+>
+  <img
+    width="456"
+    alt="tokens rendered at all sizes"
```

Comment:
We should list those sizes here, there is no textual way for users to see the different sizing.